recipes/otlhs/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set version = "1.3" %}
+
+package:
+  name: otlhs
+  version: {{ version }}
+
+source:
+  fn: otlhs-{{ version }}.tar.gz  # [not win]
+  url: https://github.com/openturns/otlhs/archive/v{{ version }}.tar.gz  # [not win]
+  sha256: 9e1f3282d3210581d9b5f62c3105b384f4e238ad3df899dcfd83b86e48b63e21  # [not win]
+
+  fn: otlhs-{{ version }}-py2.7-i686.exe  # [win32 and py27]
+  url: https://github.com/openturns/build/releases/download/v1.8rc1/otlhs-{{ version }}-py2.7-i686.exe  # [win32 and py27]
+  sha256: 1b0db8976b58db7c120fa418f306416a419dc918559ecdac04be8386d23eceb2  # [win32 and py27]
+
+  fn: otlhs-{{ version }}-py2.7-x86_64.exe  # [win64 and py27]
+  url: https://github.com/openturns/build/releases/download/v1.8rc1/otlhs-{{ version }}-py2.7-x86_64.exe  # [win64 and py27]
+  sha256: 1988ea3048faaf9862ffac7b0378c86764e424ec7ae91e5b676b5ec467f608a8  # [win64 and py27]
+
+  fn: otlhs-{{ version }}-py3.4-i686.exe  # [win32 and py34]
+  url: https://github.com/openturns/build/releases/download/v1.8rc1/otlhs-{{ version }}-py3.4-i686.exe  # [win32 and py34]
+  sha256: f4b366416601e91b25b32dd2723e7c922caf30568ef68245fa5c97855eb5ae97  # [win32 and py34]
+
+  fn: otlhs-{{ version }}-py3.4-x86_64.exe  # [win64 and py34]
+  url: https://github.com/openturns/build/releases/download/v1.8rc1/otlhs-{{ version }}-py3.4-x86_64.exe  # [win64 and py34]
+  sha256: 09374d0e63ecbf33a22df35dd8896d3b9669987e7612693ec3f41c7f90e704af  # [win64 and py34]
+
+  fn: otlhs-{{ version }}-py3.5-i686.exe  # [win32 and py35]
+  url: https://github.com/openturns/build/releases/download/v1.8rc1/otlhs-{{ version }}-py3.5-i686.exe  # [win32 and py35]
+  sha256: f761247c699e893d792d7eafbc1341fd022c081344cdd470f4867e6e7f03c77b  # [win32 and py35]
+
+  fn: otlhs-{{ version }}-py3.5-x86_64.exe  # [win64 and py35]
+  url: https://github.com/openturns/build/releases/download/v1.8rc1/otlhs-{{ version }}-py3.5-x86_64.exe  # [win64 and py35]
+  sha256: ebbb44ef635bcaa84e3ec52cce1059265d5ff20760764986573cc9722266d105  # [win64 and py35]
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - toolchain
+    - python
+    - cmake  # [not win]
+    - swig  # [not win]
+    - openturns  # [not win]
+
+  run:
+    - python
+    - openturns
+
+test:
+  imports:
+    - otlhs
+
+about:
+  home: http://openturns.org/
+  license: LGPLv3
+  summary: Optimized LHS module
```

Comment:
The `description` can be pretty verbose and multi-line. I would literally copy the entire paragraph:

```
    description: |
        OpenTURNS is a scientific C++ and Python library including an internal data
        model and algorithms dedicated to the treatment of uncertainties. The main 
        function of that library is giving to specific applications all the functionalities 
        needed to treat uncertainties in studies. Targeted users are all engineers who 
        want to introduce the probabilistic dimension in their so far deterministic studies.